src/test/java/com/skynest/LoginTest.java

```diff
@@ -9,60 +9,60 @@
 
 public class LoginTest extends BaseTest {
 
+    @DataProvider(name = "invalidCredentialCombinations")
+    public static Object[][] invalidCredentials() {
+        return new Object[][]{
+                {"invalidemail@yahoo.com", "System123"},
+                {"invalidemail@yahoo.com", "invalidpass"},
+                {"invalidemail@yahoo.com", ""},
+                {"", "System123"},
+                {"", "invalidpass"},
+                {"", ""}
+        };
+    }
+
     @Test
     void verified_user_should_be_successfully_logged_in() {
         LoginRequest body = new LoginRequest("yagaj78380@jrvps.com", "System123");
-        Response response = skyNestBackendClient.login(body);
-        response.then().statusCode(SC_OK);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_OK);
     }
 
     @Test
     void not_verified_user_should_not_be_logged_in() {
         LoginRequest body = new LoginRequest("miwix46540@jrvps.com", "System123");
-        Response response = skyNestBackendClient.login(body);
-        response.then().statusCode(SC_FORBIDDEN);
+        Response loginResponse = skyNestBackendClient.login(body);
+        loginResponse.then().statusCode(SC_FORBIDDEN);
     }
 
     @Test
     void user_should_not_be_logged_in_entering_valid_email_and_invalid_password() {
```

Comment:
This test and the following one can be merged into a single parameterized one

Reply:
Will merge them into one parameterized test in the next PR.